src/main/java/uk/gov/justice/digital/hmpps/whereabouts/controllers/CourtController.kt

```diff
@@ -49,4 +50,11 @@ class CourtController(private val courtService: CourtService) {
 
     return VideoLinkAppointmentsResponse(appointments = courtAppointments)
   }
+
+  @PostMapping(consumes = [MediaType.APPLICATION_JSON_VALUE], path = ["/video-link-bookings"])
+  @ResponseStatus(HttpStatus.CREATED)
+  @ApiOperation(value = "Create a Video Link Booking")
+  fun createVideoLinkBooking(@RequestBody @Valid videoLinkBookingSpecification: VideoLinkBookingSpecification): Long {
```

Comment:
This could be unlined: 
`  fun createVideoLinkBooking(@RequestBody @Valid videoLinkBookingSpecification: VideoLinkBookingSpecification): = courtService.createVideoLinkBooking(videoLinkBookingSpecification)`